### DISCUSSION TITLE
GitHub Action

### DISCUSSION THREAD

**Discussion by alessiodionisi** [FiloSottile/age#362] [General]
Hello! Some time ago I made an action for GitHub which installs `age` in the path. For example it can be used with SOPS and Terraform.

I think it might be useful so I decided to make a post here in the discussions 😄 

https://github.com/marketplace/actions/setup-age

**Comment by vsajip** [comment]
It didn't work for me yesterday on Windows. I had to use `choco install` instead.

**Reply by alessiodionisi** [reply]
I'll take a look tomorrow 🤔
